TheHub/TheHub.WebApp/Controllers/ReviewController.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using TheHub.Library.Interfaces;
+using TheHub.Library.Model;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace TheHub.WebApp.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ReviewController : ControllerBase
+    {
+        private readonly IReviewRepo _reviewRepository;
+
+        public ReviewController(IReviewRepo Repository)
+        {
+            _reviewRepository = Repository;
+        }
+        // GET: api/<ReviewController>
+        [HttpGet]
+        public IEnumerable<string> Get()
+        {
+            return new string[] { "value1", "value2" };
+        }
+
+        // GET api/<ReviewController>/5
+        [HttpGet("{id}")]
+        public string Get(int id)
+        {
+            return "value";
+        }
+
+        // POST api/<ReviewController>
+        [HttpPost("CreateReview")]
+        public IActionResult CreateReveiw([FromBody] Review review)
+        {
+            Review newR = new Review
+            {
+                Rating = review.Rating,
+                MediaId = review.MediaId,
+                UserId = review.UserId,
+                Content = review.Content
+            };
+            _reviewRepository.Add(newR);
+
+            var newReview = _reviewRepository.GetById(newR.ReviewId);
+
+            return Created(""+review.ReviewId+"",newReview);
+        }
+
+        // PUT api/<ReviewController>/5
+        [HttpPut("{id}")]
+        public void Put(int id, [FromBody] string value)
+        {
+        }
+
+        // DELETE api/<ReviewController>/5
+        [HttpDelete("{id}")]
+        public void Delete(int id)
+        {
+        }
+    }
+}
```

Comment:
we probably shouldn't merge master with incomplete code (e.g put and delete methods have nothing in them)